ButtplugControlLibrary/ButtplugTabControl.xaml.cs

```diff
@@ -119,7 +119,7 @@ public void InitializeButtplugServer()
                 }
                 catch (PlatformNotSupportedException e)
                 {
-                    _ravenClient.Capture(new SentryEvent(e));
+                    SendExceptionToSentry(e);

```

Comment:
Ehhhhhhhh let's just turn this into a WARN level log message. I don't really see us needing reports of this showing up on sentry, since we handle the case where it happens now (I put this in when it was still causing crashes)